command/genesis/apex_params.go

```diff
@@ -9,7 +9,7 @@ import (
 
 const (
 	apexBridgeFlag             = "apex"
-	apexBridgeFlagDefaultValue = true
+	apexBridgeFlagDefaultValue = false
```

Comment:
do not disable default apex-bridge behavior. 
Even better rename this flag as `apex-configuration` and make it enum field. Process different configurations with switch 

```
switch ... {
case ApexConfigurationDefaultBridge:
	chainConfig.Params.Forks.RemoveFork(chain.Governance).RemoveFork(chain.London)
	chainConfig.Params.BurnContract = nil
case ApexConfigurationNexus:
         chainConfig.Params.Forks.
		RemoveFork(chain.Governance).
		RemoveFork(chain.EIP3855).
		RemoveFork(chain.Berlin).
		RemoveFork(chain.EIP3607)
       	chainConfig.Params.Forks.RemoveFork(chain.Governance).RemoveFork(chain.London)
	chainConfig.Params.BurnContract = map[uint64]types.Address{
		0: types.ZeroAddress,
	}
	chainConfig.Genesis.GasLimit = 0x500000
}